docs/Setup.md

```diff
@@ -71,6 +71,9 @@ Start by making sure the command line with Gradle works first, because **all the
 1. Reboot your computer, which will ensure that Gradle and Kotlin daemons are completely killed and relaunched
 1. Delete the global Gradle cache under `~/.gradle/caches`
 1. If adding a new dependency or updating a dependency, a warning that a dependency cannot be found may indicate the Maven repository restrictions need adjusting
+1. If included build (like buildSrc) contains different kotlin version (embedded from gradle) than other modules 
```

Comment:
We could also link the follow-up issue here in the known issues. Thank you for taking care of it.